src/main/kotlin/no/nav/familie/tilbake/common/ContextService.kt

```diff
@@ -17,30 +19,40 @@ object ContextService {
     }
 
     fun hentPåloggetSaksbehandler(defaultverdi: String?): String {
-        return Result.runCatching { SpringTokenValidationContextHolder().tokenValidationContext }
+        return Result.runCatching { SpringTokenValidationContextHolder().getTokenValidationContext() }
             .fold(
                 onSuccess = {
-                    return it.getClaims("azuread")?.get("NAVident")?.toString()
+                    return it.getAzureadClaimsOrNull()?.get("NAVident")?.toString()
                         ?: defaultverdi
                         ?: throw Feil("Ingen defaultverdi for bruker ved maskinelt oppslag")
                 },
                 onFailure = { defaultverdi ?: throw Feil("Ingen defaultverdi for bruker ved maskinelt oppslag") },
             )
     }
 
+    // TODO Fjern hack for midlertidig simulere oppførsel fra tidligere versjon av no.nav.security.token.support.core
+    @Deprecated("Ikke bruk! kommer til å byttes ut med versjon som kaster exception ved manglende issuer claims i neste versjon av tilbake")
+    private fun TokenValidationContext.getAzureadClaimsOrNull(): JwtTokenClaims? {
+        return try {
+            this.getClaims("azuread")
+        } catch (e: IllegalArgumentException) {
+            null
+        }
+    }
+
```

Comment:
Alternativ - kanskje bedre/noe vi kan leve med? Bruke strict/defaultverdi==null dere vi kaller funksjon?  
 
` private fun TokenValidationContext.getAzureadClaimsOrNull(kastFeilHvisAzureadTokenMangler: Boolean): JwtTokenClaims? {
        return try {
            this.getClaims("azuread")
        } catch (e: IllegalArgumentException) {
            val kastFeil = kastFeilHvisAzureadTokenMangler && e.message?.contains("No token found for issuer azuread") == true
            when {
                kastFeil -> throw e
                else -> null
            }
        }
    }`